### ISSUE TITLE
Implement production database

### ISSUE THREAD

**Issue by henryk3dp** [SNDjango/server#19]
We should implement a production database. 
Travis already uses Postgres. you can view the config [here](https://github.com/SNDjango/server/blob/devel/snd/snd/settings.py#L80)

we can keep the sqlite database for testing and control which database gets used by environment variables, but using a production database should be possible.

**Comment by henryk3dp** [comment]
depends on: #17

**Comment by atamankina** [comment]
This is the first Database diagram. This version is valid for the basic functionality. The model should be extended for further functionality.
Will put the Visio source to Slack, as Github doesn't want to save it here.

**Comment by henryk3dp** [comment]
@atamankina  implementing the models should be a different issue or be done as needed. i have for example implemented the ContentItem model in #20 and will push it later today.
This issue only concerns database configuration.

**Comment by atamankina** [comment]
@Teszko thanks for the suggestion, I'll create another ticket for that.

**Comment by atamankina** [comment]
To be tested on the deployment server.
